content/_includes/post.macros.njk

```diff
@@ -355,8 +355,10 @@ params:
 
     {{ client(post.data.client) }}
 
-    {% if dateformat %}
-      {{ pubdate(post.date, post.data.end, dateformat) }}
+    {% if not post.data.events %}
+      {% if dateformat %}
+        {{ pubdate(post.date, post.data.end, dateformat) }}
+      {% endif %}
```

Comment:
I left a comment in Trello about whether we would want to also use this approach on the Event single view or if the published date is OK when not in a list.

Reply:
I don't have strong feelings on this one way or the other. I think it's much less confusing on the detail page.